src/_singletons/ContractDetailsSingleton.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+import Modal from '../containers/Modal';
+import ContractDetailsContainer from '../contract-details/ContractDetailsContainer';
+import ContractDetailsSingletonSelector from './ContractDetailsSingletonSelector';
+
+@connect(ContractDetailsSingletonSelector)
+export default class ContractDetailsSingleton extends Component {
```

Comment:
OK, container already being used, I'll come up with another name.

And fix the test